web/app/scripts/views/record/add-edit-partial.html

```diff
@@ -42,7 +42,7 @@
                             </div>
                         </div>
                         <div class="col-md-12">
-                            <div class="map" leaflet-map driver-embed-map style="height:300px;"></div>
+                            <div class="map" leaflet-map driver-embed-map editable="true"></div>
```

Comment:
Also, it is possible to use something like: ng-if="ctl.record" to make sure the directive doesn't render before the data is available. Not suggesting you should change that here, just letting you know since I've found it useful.